interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
I am assuming the current rule you have is basically the same as the 1 current validation rule on this form(let me know if this is not the case).